markdown-export/render-rules.ts

```diff
@@ -16,6 +16,44 @@ interface TableCell {
     align: string | undefined;
 }
 
+function parseAndFormatDate(str: string): string {
```

Comment:
Both these functions should be moved to `render-helpers.ts` to only have the rules themselves in this file.

Reply:
Have moved `formatYAMLString` to `render-helpers.ts`. No longer need `parseAndFormatDate` so have removed completely.